Still failing for me.
```
/tmp/nix-build-gplates-2.0.0.drv-0/gplates-2.0.0/src/property-values/ProxiedRasterResolver.cc:119:1:   required from here
include/boost/optional/detail/optional_reference_spec.hpp:83:8: error: 'const GPlatesGui::Mipmapper<GPlatesPropertyValues::RawRasterImpl<signed char, GPlatesPropertyValues::RawRasterDataPolicies::WithData, GPlatesPropertyValues::RawRasterStatisticsPolicies::WithStatistics, GPlatesPropertyValues::RawRasterNoDataValuePolicies::WithNoDataValue>, void>* boost::optional<const GPlatesGui::Mipmapper<GPlatesPropertyValues::RawRasterImpl<signed char, GPlatesPropertyValues::RawRasterDataPolicies::WithData, GPlatesPropertyValues::RawRasterStatisticsPolicies::WithStatistics, GPlatesPropertyValues::RawRasterNoDataValuePolicies::WithNoDataValue>, void>&>::ptr_' is private
include/boost/optional/detail/optional_reference_spec.hpp:97:82: error: within thiscontext
make[2]: *** [src/property-values/CMakeFiles/property-values.dir/build.make:1215: src/property-values/CMakeFiles/property-values.dir/ProxiedRasterResolver.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1007: src/property-values/CMakeFiles/property-values.dir/all] Error 2
make: *** [Makefile:128: all] Error 2
builder for ‘/nix/store/cxfgmg9aw6bmdzv4dx7q06s1b6g6dns8-gplates-2.0.0.drv’ failed with exit code 2
error: build of ‘/nix/store/cxfgmg9aw6bmdzv4dx7q06s1b6g6dns8-gplates-2.0.0.drv’ failed
```

--- a/pkgs/applications/science/misc/gplates/default.nix
+++ b/pkgs/applications/science/misc/gplates/default.nix
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, qt4, qwt6_qt4, mesa, glew, gdal_1_11, cgal
-, proj, boost159, cmake, python2, doxygen, graphviz, gmp }:
+, proj, boost, cmake, python2, doxygen, graphviz, gmp }:
 
 stdenv.mkDerivation rec {
   name = "gplates-${version}";
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    qt4 qwt6_qt4 mesa glew gdal_1_11 cgal proj boost159 cmake python2
+    qt4 qwt6_qt4 mesa glew gdal_1_11 cgal proj boost cmake python2
     doxygen graphviz gmp
   ];
 

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -17467,7 +17467,10 @@ with pkgs;
 
   fityk = callPackage ../applications/science/misc/fityk { };
 
-  gplates = callPackage ../applications/science/misc/gplates { };
+  gplates = callPackage ../applications/science/misc/gplates {
+    boost = boost160;
+    cgal = cgal.override { boost = boost160; };
+  };
 
   gravit = callPackage ../applications/science/astronomy/gravit { };
 

